packages/lab/src/stepped-tracker/SteppedTracker.css

```diff
@@ -1,17 +1,41 @@
 .saltSteppedTracker {
-  margin: 0;
-  padding: 0;
-  text-indent: 0;
-  list-style-type: none;
-  display: flex;
+  grid-area: SteppedTracker;
   width: 100%;
-  position: relative;
+  height: 100%;
+
+  padding: var(--salt-focused-outlineWidth);
+  margin: calc(-1 * var(--salt-focused-outlineWidth));
+  list-style-type: none;
+  box-sizing: content-box;
```

Comment:
Make sure to put `box-sizing` in other CSS where necessary.

Reply:
In Accordion you guys have used negative offset for the focus outline on the button part, which is not a bad solution. Don't know if it will pass with our designers and a11y experts, will check with them

Reply:
Done!